docker/Dockerfile_centos_deps

```diff
@@ -35,6 +35,8 @@ ENV AV_DEV=/opt/AliceVisionDeps_git \
 #
 RUN $YUM_INSTALL centos-release-scl-rh && \
     $YUM_INSTALL yum-utils && \
+    $YUM_INSTALL python3 && \
+    $YUM_INSTALL gcc python3-devel && \
```

Comment:
Why "gcc" here? We install "devtoolset-10-toolchain" 2 lines below.

Reply:
"gcc" is needed to build "psutil". If it is missing, the following error message occurs, requesting both "gcc" and "python3-devel". "devtoolset-10-toolchain" does not seem to be enough.

![image](https://github.com/alicevision/AliceVision/assets/11963329/f9c8d541-0de9-4feb-bd1d-4c26acfd0a8d)

Reply:
"gcc" can indeed be removed: it was required because psutil couldn't be installed since python3-devel was missing, and it thus needed to be built (with gcc).

This is addressed in #1564.